src/utils/stripLocationInfo.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Returns a copied version of the node recieved without location attributes.
+ * @param  {Object}  node
+ * @return {Object}  node
+ */
+export default function stripLocationInfo(node) {
```

Comment:
I have removed stripLocationInfo.js and used isEqualAst instead: d1cfb7a590678c0ff04614b05e57cc27115006ed